app/notification_push.go

```diff
@@ -212,13 +233,27 @@ func (a *App) ClearPushNotificationSync(userId string, channelId string) {
 	for _, session := range sessions {
 		tmpMessage := *model.PushNotificationFromJson(strings.NewReader(msg.ToJson()))
 		tmpMessage.SetDeviceIdAndPlatform(session.DeviceId)
-		a.sendToPushProxy(tmpMessage, session)
+		err := a.sendToPushProxy(tmpMessage, session)
+		if err != nil {
+			mlog.Error(fmt.Sprintf("Error sending push notification: UserId=%v SessionId=%v message=%v", session.UserId, session.Id, err.Error()), mlog.String("user_id", session.UserId))
+		}
 	}
 }
 
 func (a *App) ClearPushNotification(userId string, channelId string) {
+	notificationId := model.NewId()
```

Comment:
this is tricky, including an id in the clear notification should be use to clear a specific notification. see https://github.com/wix/react-native-notifications/blob/master/docs/advancedIos.md

But I guess we want to use it for something else, so lets leave it as is

Reply:
This ID refers to the notification itself (a notification can be sent to multiple devices, and each one will receive the same `notificationId`) I'm not 100% sure if is something that we want. Actually this `notificationId` can be rename to other thing to get it less confusing, something like `notificationTrackingId`

Reply:
Every id should be unique, even if the recipient is the same user

Reply:
Yes, the thing is, if we want to track before the enqueue, we need an id there which is share to each notification sent to the same user in different devices, so we can add another extra ID to each notification per device.